websites/R/Reaper Scans/presence.ts

```diff
@@ -3,87 +3,80 @@ const presence = new Presence({
 	}),
 	elapsed = Math.floor(Date.now() / 1e3);
 
+enum Assets {
+	logo = "https://i.imgur.com/tPMtbjL.png",
+	search = "https://i.imgur.com/08wjeL0.png",
+	read = "https://i.imgur.com/Gglu4Tk.png",
+	view = "https://i.imgur.com/BMzYEcO.png",
```

Comment:
Capitals. Also change the mentions of this
```suggestion
	Logo = "https://i.imgur.com/tPMtbjL.png",
	Search = "https://i.imgur.com/08wjeL0.png",
	Read = "https://i.imgur.com/Gglu4Tk.png",
	View = "https://i.imgur.com/BMzYEcO.png",
```